cmd/daemon/flux-notification-controller/server.go

```diff
@@ -0,0 +1,15 @@
+package flux_notification_controller
```

Comment:
A nit, but usually we don't have multi word package names. I'm at a loss for coming up with a better name right now through, so let's keep it as is 🙈

Reply:
flux2noticatons would be an alternative? :)

Reply:
Yeah at least that avoids the `_` 👍